test/src/base/MooseTestApp.C

```diff
@@ -29,6 +30,12 @@ MooseTestApp::validParams()
                                    false,
                                    "Call getRestartableDataMap with a bad name.");
 
+  // Flag for turning how EigenProblem output eigenvalues
+  params.addCommandLineParam<bool>("output_inverse_eigenvalue",
+                                   "--output-inverse-eigenvalue",
+                                   false,
+                                   "True to let EigenProblem output inverse eigenvalue.");
+
```

Comment:
#endif

Reply:
Theoretically speaking this code works without SLEPc. Do you still want me to put the if guard here?

Reply:
OK, I will add that later in my PR.